includes/templates/single-web-story.php

```diff
@@ -26,97 +26,7 @@
 
```

Comment:
I think not called `wp_head`, `wp_body_open` and `wp_footer` on this template is issue here. Many plugins use to hook in on the FE to add tracking, like google analytics or tracking pixels. This may upset users that these scripts do not load as expected.

Reply:
It is worth noting that we've never had these hooks for stories.

We need to be very careful about not letting any plugins break the story because they can add custom JavaScript and otherwise break AMP compatibility.

Sure, we can add our custom hooks, but the default ones are too risky.

Reply:
I think maybe we do need add custom versions of these hooks, so plugins like Yoast can hook into them.